$NetBSD: patch-aq,v 1.1 2007/03/17 13:25:46 rillig Exp $

PendingTimers is only used in timer.c, so there is no need for an
"extern" declaration.

--- include/struct.h.orig	2003-04-11 03:09:07.000000000 +0200
+++ include/struct.h	2007-03-17 14:20:26.000000000 +0100
@@ -1064,7 +1064,6 @@ struct	timeval	time;
 	int	delete;
 }	TimerList;
 
-extern TimerList *PendingTimers;
 typedef struct nicktab_stru
 {
 	struct nicktab_stru *next;
